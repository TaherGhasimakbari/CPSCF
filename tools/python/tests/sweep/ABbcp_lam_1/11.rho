 format  1  0
dim                 
                   1
crystal_system      
          'lamellar'
N_cell_param        
                   1
cell_param          
    1.4840011410E+00
group_name          
                '-1'
N_monomer           
                   2
N_star              
                  17
  3.900000002235E-01  6.099999997765E-01       0     1
  2.997902145580E-01 -2.997902145487E-01       1     2
  5.715020500378E-02 -5.715020497704E-02       2     2
 -1.148477954676E-02  1.148477955833E-02       3     2
 -8.814719746472E-03  8.814719753241E-03       4     2
 -8.663694711190E-04  8.663694682604E-04       5     2
  8.247508036368E-04 -8.247508058364E-04       6     2
  3.104240940577E-04 -3.104240938354E-04       7     2
 -2.003282595256E-05  2.003282664735E-05       8     2
 -4.201547046516E-05  4.201547079990E-05       9     2
 -8.190907916060E-06  8.190907685009E-06      10     2
  2.966547857865E-06 -2.966548324828E-06      11     2
  1.738384483885E-06 -1.738384619344E-06      12     2
  8.241256092361E-08 -8.241229479602E-08      13     2
 -1.895414198368E-07  1.895416842956E-07      14     2
 -4.963153468017E-08  4.963146880300E-08      15     2
  1.177455548340E-08 -1.177474644176E-08      16     1
