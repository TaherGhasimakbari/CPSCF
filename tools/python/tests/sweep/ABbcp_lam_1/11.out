 format  1  0

MONOMERS            
N_monomer           
                   2
kuhn                
    1.0000000000E+00    1.0000000000E+00

CHAINS              
N_chain             
                   1
N_block             
                   2
block_monomer       
                   1                   2
block_length        
    3.9000000022E-01    6.0999999978E-01

COMPOSITION         
ensemble            
                   0
phi_chain           
    1.0000000000E+00

INTERACTION         
interaction_type    
               'chi'
chi                 
    1.5000000011E+01

UNIT_CELL           
dim                 
                   1
crystal_system      
          'lamellar'
N_cell_param        
                   1
cell_param          
    1.4840011410E+00

DISCRETIZATION      
ngrid               
                  32
chain_step          
    1.0000000000E-02

FILE_PREFIXES       
input_prefix        
               'in.'
output_prefix       
              'ref/'

BASIS               
group_name          
                '-1'

ITERATE             
max_itr             
                  20
error_max           
    1.0000000000E-08
domain              
                   T
itr_algo            
                'NR'
N_cut               
                  17

SWEEP               
s_max               
    2.0000000000E+01
d_block_length      
   -1.0000000000E-02    1.0000000000E-02
end_increments      

FINISH              

THERMO              
f_Helmholtz         
    2.3777398784E+00
f_homo              
    2.5685000034E+00
pressure            
    3.1682282895E+00
mu_chain            
    5.5459681679E+00
stress              
    1.1906856658E-11

STATISTICS          
N_star              
                  17
Final Error         
    1.3088475859E-09
Iterations          
                   6
Basis Time          
    2.6167106628E-01
SCF Time            
    4.7572135925E-02
